README.md

```diff
@@ -207,10 +208,11 @@ The `NuxtAuthHandler` accepts [all options that NextAuth.js accepts for its API
 ##### Example with two providers
 
 Here's what a full config can look like, wee allow authentication via a:
-- Github Oauth flow,
+- Github Oauth flow
 - a username + password flow (called `CredentialsProvider`)
+- Strapi JWT example with the `CredentialsProvider`
 
-Note that the below implementation of the credentials provider is flawd and mostly copied over from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials) in order to give a picture of how to get started with the credentials provider:
+Note that the below implementation of the credentials provider is flawed and mostly copied over from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials) in order to give a picture of how to get started with the credentials provider:
```

Comment:
Let's move this below the example code in the following form:
```
Note: The above credential-provider example is taken over in part from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials). It is _not_ considered safe for production usage and you would need to adapt it further, e.g., by calling another service that provides authentication, such as [strapi](#example-with-a-custom-strapi-jwt-provider)
```

Reasoning: the code already specifies that it is unsafe + the code contains parts that are not from the nextauth-docs example, like the github oauth part + the nuxt virtual imports.